src/app/containers/Article/index.jsx

```diff
@@ -11,50 +11,77 @@ import ArticleMain from '../ArticleMain';
 import ErrorMain from '../ErrorMain';
 import nodeLogger from '../../helpers/logger.node';
 import ConsentBanner from '../ConsentBanner';
+import { GhostWrapper, GridItemConstrainedMedium } from '../../lib/styledGrid';
 
 const logger = nodeLogger(__filename);
 
-/*
-  [1] This handles async data fetching, and a 'loading state', which we should look to handle more intelligently.
-*/
-const ArticleContainer = ({ loading, error, data, bbcOrigin }) => {
-  if (loading) return 'Loading...'; /* [1] */
-  if (error) {
-    logger.error(error);
-    return 'Something went wrong :(';
-  }
+const ArticleWrapper = ({ bbcOrigin, children, isAmp, service }) => (
+  <Fragment>
+    <GlobalStyle />
+    <RequestContextProvider
+      platform={isAmp ? 'amp' : 'canonical'}
+      bbcOrigin={bbcOrigin}
+    >
+      <ServiceContextProvider service={service}>
+        <Helmet>
+          <link rel="manifest" href={`/${service}/articles/manifest.json`} />
+        </Helmet>
+        <ConsentBanner />
+        <HeaderContainer />
+        {children}
+      </ServiceContextProvider>
+    </RequestContextProvider>
+  </Fragment>
+);
+
+ArticleWrapper.propTypes = {
+  bbcOrigin: string,
+  children: node.isRequired,
+  isAmp: bool.isRequired,
+  service: string.isRequired,
+};
+
+ArticleWrapper.defaultProps = {
+  bbcOrigin: null,
+};
 
+const ArticleContainer = ({ loading, error, data, bbcOrigin }) => {
   if (data) {
```

Comment:
This PR seems to swap the execution order of `loading`, `error` and `data`- was this on purpose? 

The new logic seems that if `data` return as a falsey value then we wouldn't hit `error` because it's all nested inside of this `if (data) {` 🤔

Reply:
@pjlee11 Yes this order has been changed - this is because now our error page and loading page require the `RequestContext` for the Consent Banners, we need to have `isAmp` `bbcOrigin` & `service` passed to these states.

Reply:
So in the case we don't get a `data` object we would return `null` and the user would see a completely blank page? I would want both Product and UX to sign off on this as I'd assume it's not something we'd want within our error boundaries. Could we not have a "default" error page?

Reply:
I agree that we should ensure we cover the scenario where we do not have those values.  The simplest option would be to have defaults:

| prop | default|
| ----- | ---- |
| isAmp | `false` |
| service | `news` |
| status | `500` |
| bbcOrigin | `https://www.bbc.co.uk` |

However, I don't believe we should have an error page that defaults to a different service than the one we are currently on. e.g. if we're on `persian`, we shouldn't show a `news` 500 error page. 

I believe the changes we'd need to make, include the separation of `isAmp`, `service` & `status` from articleData in `getInitialData` so that `App/App.jsx` can call `getInitialData` for _just_ the article data and get the other values from another function.

Reply:
My opinion would be that we should add a default in this PR to mitigate potentially showing a blank page, and because we are only in production with `news` the default can be logically set to `news`. We should also open an issue labelled with `must` and `v1.1` to properly handle the separation you have described above

Reply:
Therefore, in this PR have something like:
```
if (loading) { ... }
if (error) { ErrorPage //using default params }
if (data) { 
  try {
    render()
  } catch {
    ErrorPage // using service params
  }
}
```